interfaces/cython/cantera/reactor.pyx

```diff
@@ -727,10 +727,10 @@ cdef class MassFlowController(FlowDevice):
 
 cdef class Valve(FlowDevice):
     r"""
-    In Cantera, a `Valve` is a flow devices with mass flow rate that is a
+    In Cantera, a `Valve` is a flow device with mass flow rate that is a
     function of the pressure drop across it. The default behavior is linear:
 
-    .. math:: \dot m = K_v (P_1 - P_2)
```

Comment:
If you think a multiplication symbol is really needed here (not sure I agree), then (a) there are a lot of other similar cases that should be addressed and (b) I'd consider using `\cdot` instead of `*`.